app/frontend/components/domains/permit-application/edit-permit-application-screen.tsx

```diff
@@ -93,7 +93,7 @@ export const EditPermitApplicationScreen = observer(({}: IEditPermitApplicationS
   const { permitTypeAndActivity, formJson, number, isSubmitted } = currentPermitApplication
 
   return (
-    <Box as="main" overflow="auto" h="full">
+    <Box as="main" overflow="hidden" h="full">
```

Comment:
@bkmaibach `<Box as="main"  h="1px" minH="1px" flex={1}>`, This should keep the footte